frontend/views/pages/Payments.vue

```diff
@@ -63,6 +63,20 @@ page(
       )
 
       .tab-section
+        banner-simple(severity='info')(
+          v-if='!isCloseToDistributionTime'
```

Comment:
This condition should be inverted:

`!isCloseToDistributionTime` => `isCloseToDistributionTime`